### PR TITLE
2.1.5 release notes

### PR DESCRIPTION
**Pull request by danielmewes**
@mglukhovsky could you review please?

### CODE REVIEW

File: NOTES.md [hunk 1]
```diff
@@ -1,9 +1,60 @@
...
+
+--
+
 # Release 2.1.4 (Forbidden Planet)
 
 Released on 2015-09-16
 
 Bug fix release
 
+### Compatiblity ###
```
Comment (danielmewes) [inline NOTES.md]: This is slightly unorthodox, but I decided to retrofit the 2.1.4 release notes to have the same format and compatibility warning as the ones for 2.1.2 and 2.1.3 (and now 2.1.5).

File: NOTES.md [hunk 2]
```diff
@@ -1,9 +1,60 @@
...
+### Bug fixes ###
+
+* Made the build system compatible with OS X El Capitan (#4602)
+* Fixed spurious "Query terminated by `rethinkdb.jobs` table" errors (#4819)
+* Fixed an issue that caused changefeeds to keep failing after a table finished
+  reconfiguring (#4838)
+* Fixed a race condition that resulted in a crash with the message
+  `std::terminate() called without any exception.` when losing a cluster connection
+  (#4878)
+* Fixed a segmentation fault in the `mark_ready()` function that could occurr when
```
Comment (danielmewes) [inline NOTES.md]: -> occur :-D

File: NOTES.md [hunk 3]
```diff
@@ -1,9 +1,60 @@
...
+* Fixed an `Assertion failed: [ptr_]` error when reconfiguring tables quickly with a
+  debug-mode binary (#4871)
+* Improved the detection of unsupported values in `r.js` functions to avoid a
+  `Guarantee failed: [!key.IsEmpty() && !val.IsEmpty()]` crash in the worker process
+  (#4879)
+* Fixed an unitialized data access issue on shutdown (#4918)
+
+### Performance improvements ###
+
+* Improved the performance of `getAll` queries which fetch multiple keys at once (#1526)
```
Comment (mglukhovsky) [inline NOTES.md]: which -> that

File: NOTES.md (hunk 1 above)
Comment (mglukhovsky) [inline NOTES.md]: :+1:

**Comment by mglukhovsky** [comment]
OK, left comments.

**Comment by danielmewes** [comment]
Thanks @mglukhovsky. I'll update the PR tomorrow.

**Comment by danielmewes** [comment]
Updated the PR.

**Pull request merged**